packages.yml

```
@@ -1,3 +1,6 @@
 packages:
-- package: fivetran/stripe_source
-  version: [">=0.14.0", "<0.15.0"]
\ No newline at end of file
+# - package: fivetran/stripe_source
+#   version: [">=0.14.0", "<0.15.0"]
+- git: https://github.com/fivetran/dbt_stripe_source.git
+  revision: bug/does-table-exist
+  warn-unpinned: false 
```

```suggestion
- package: fivetran/stripe_source
  version: [">=0.14.0", "<0.15.0"]
```
Update before merge.